test/shared/filters.js

```diff
@@ -13,7 +13,7 @@ exports.test = function(name) {
       var str = fs.readFileSync('test/fixtures/' + name + '/filters.' + name).toString();
 
       var locals = { user: user, filters: { toupper: function(object) {
-        return object.toUpperCase();
+        return String(object).toUpperCase();
```

Comment:
The general test should not be changed here / the `object` should actually be a string or a String object.